packages/slice-machine/server/src/api/auth/post.ts

```diff
@@ -0,0 +1,34 @@
+import * as t from "io-ts";
+import { fold } from "fp-ts/Either";
+import { FileSystem } from "slicemachine-core";
+
+const AuthRequest = t.type({
+  email: t.string,
+  cookies: t.array(t.string),
+});
+
+type AuthRequest = { email: string; cookies: string[] };
```

Comment:
You can do this, it's cooler and we never redefine twice the type :)
```suggestion
 type AuthRequest = t.TypeOf<typeof AuthRequest>
```

Reply:
Ohhh love that

Reply:
Fixed :)